### DISCUSSION TITLE
Welcome to pudb Discussions!

### DISCUSSION THREAD

**Discussion by inducer** [inducer/pudb#474] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mvanderkamp** [comment]
Hey! Is this a new feature? I haven't seen it on Github before!

Anyway, I picked up `pudb` roughly a year and a half ago and use it for all my python debugging. I've really enjoyed contributing general fixes and improvements to make life easier for myself and everyone else who uses this tool! I enjoy digging into and sprucing up existing code and features, and open source is a refreshing space to do exactly that.

**Reply by inducer** [reply]
I think it's semi-new? Not sure. Since I turned off the mailing lists Wednesday (https://lists.tiker.net/), I figured there should be some sort of replacement. :)
